packages/Webkul/Shop/src/Resources/views/customers/account/profile/edit.blade.php

```diff
@@ -208,22 +208,27 @@ class="mb-4"
             </x-shop::form.control-group.error>
         </x-shop::form.control-group>
 
-        <x-shop::form.control-group class="mb-4">
-            <x-shop::form.control-group.control
+        <div class="select-none items-center flex gap-[6px] mb-4">
+            <input
                 type="checkbox"
                 name="subscribed_to_news_letter"
-                :checked="$customer->subscribed_to_news_letter"
-            >
-                <span class="select-none text-[16] text-[#7d7d7d] max-sm:text-[12px]">
-                    @lang('shop::app.customer.signup-form.subscribe-to-newsletter')
-                </span>
-            </x-shop::form.control-group.control>
-
-            <x-shop::form.control-group.error
-                control-name="subscribed_to_news_letter"
-            >
-            </x-shop::form.control-group.error>
-        </x-shop::form.control-group>
+                id="is-subscribed"
+                class="hidden peer"
+                onchange="switchVisibility()"
```

Comment:
Where "switchVisibility" method is defined

Reply:
fixed